### ISSUE TITLE
Add cryostat camera view to BioXAS Side

### ISSUE THREAD

**Issue by helfrij** [acquaman/acquaman#1781]
There is a camera being installed in the SOE of BioXAS Side that would be used to help with alignment. The camera hasn't been installed yet, but it is planned to be installed soon. 
- One trick that's been discussed is making sure the camera view is only actively updating/refreshing when it is actually being used (ie it's view is the current view). 
- I expect that there is existing framework code that embeds a camera feed already, this implementation would just be mirroring existing usages elsewhere.

**Comment by helfrij** [comment]
The camera has been installed, but may handle data differently that other camera views in Acquaman. Waiting on Ian C. for more information.

**Comment by helfrij** [comment]
On hold. The camera output is trickier to get ahold of than expected.

**Comment by helfrij** [comment]
This camera is a Basler camera, and doesn't support .mjpg output. For now, this camera won't be added to the BioXAS applications. Should that change in the future, it will be handled as a separate issue.
